tests/src/RunBenchmarks/RunBenchmarks.cmd

```diff
@@ -0,0 +1,20 @@
+setlocal
+
+set BENCHMARK_ROOT_DIR=\git\repos\DotNet\coreclr\bin\tests\Windows_NT.x64.Release\JIT\Performance\CodeQuality
+set BENCHMARK_HOME=\git\repos\DotNet\coreclr\tests\src\RunBenchmarks
+set BENCHMARK_HOST=%BENCHMARK_HOME%\bin\Debug\dnxcore50\RunBenchmarks.exe
+set BENCHMARK_HOST=%BENCHMARK_HOME%\bin\Debug\desktop\RunBenchmarks.exe
+set BENCHMARK_RUNNER=-runner CoreRun.exe
+
+start DHandler.exe
```

Comment:
taskkill comes with Windows. Maybe we should look into open sourcing DHandler? Agree about setting COMPlus_NoGuiOnAssert